README.md

```diff
@@ -115,6 +115,23 @@ _build/prod/rel/mongoose_push/bin/mongoose_push console
 
 Yeah, I know... It crashed. Running this service is fast and simple but unfortunately you can't have push notifications without properly configured `FCM` and/or `APNS` service. So, lets configure it!
 
+### Running tests
+
+Setup FCM and APNS mocks first:
+```bash
+$ docker-compose -f test/docker/docker-compose.yml up -d
+```
+
+Run tests:
+```bash
+$ mix test
```

Comment:
```suggestion
$ mix do certs.dev, test
```

Reply:
Or maybe even split that with a remark that `certs.dev` is only needed once.

Reply:
yeah, right - I forgot about certs